### DISCUSSION TITLE
should optimizeFonts add preconnect link as well?

### DISCUSSION THREAD

**Discussion by stefanprobst** [vercel/next.js#17371] [Help]
i've been using the experimental `optimizeFonts` option, and it's been working great. one thing i'm unclear about is if i should additionally add a `<link rel"preconnect">` to "fonts.gstatic.com"? or if this something `optmizeFonts` *should* be doing but doesn't? or if this is actually detrimental to perf and is intentionally *not* done by the `optimizeFonts` option?

**Comment by lfades** [comment]
cc @prateekbh

**Comment by prateekbh** [comment]
Hi! The investigation to pick an option between pre-connect vs preload is pending on our end.
Will update this thread as soon as we have a better answer

**Comment by TommySorensen** [comment]
@stefanprobst I think you should still include the `preconnect` since it's only the css that's put into the DOM and not the actual font file. So you will still see an request to the `fonts.gstatic.com` site to receive the file
